express-delete/index.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  const gradesArr = [];
+  gradesArr.push(grades);
```

Comment:
bump

Reply:
Sorry for missing that! I believe this revision resolves the issue.

<img width="1431" alt="Screen Shot 2021-10-08 at 5 18 21 PM" src="https://user-images.githubusercontent.com/74282257/136636902-44dae4bf-2756-4426-8db5-c0202d7f28c1.png">